public/static/locales/en/form-builder.json

```diff
@@ -262,6 +262,7 @@
   "settingsResponseNotePublished": "To change the response delivery destination.",
   "settingsResponseTitle": "Response delivery destination",
```

Comment:
Is this a person / email address?

Reply:
In this case it's an email address

<img width="746" alt="Screen Shot 2023-02-17 at 8 26 11 AM" src="https://user-images.githubusercontent.com/62242/219664787-2553269b-d023-43df-a5e9-624158ee95c6.png">